src/dlgAboutDialog.cpp

```diff
@@ -81,7 +81,7 @@ dlgAboutDialog::dlgAboutDialog(QWidget * parent) : QDialog(parent)
         }
 
         // Repeat for other text, but we know it will fit at given size
-        QString sourceCopyrightText = QChar( 169 ) % QString( " Heiko K" ) % QChar( 246 ) % QString( "hn 2008-" ) % QString(__DATE__).mid(7);
+        QString sourceCopyrightText = QChar( 169 ) % QString( " Mudlet devs 2008-" ) % QString(__DATE__).mid(7);
```

Comment:
"Mudlet devs", that's a collective of the Mudlet developers. Copyright is stated on each file anyway that people contribute to. That page doens't go into much more detail about it - so do we need to do anything?